src/components/SiteHeader/siteHeader.scss

```diff
@@ -69,9 +73,21 @@ html:not([class=""]) {
         }
       }
     }
-  }
 
-  
+    .a-logo__svg--100 {
+      @include breakpoint("medium+") {
+        transform: scale(0.75) translate(0%, -10%);
+      }
```

Comment:
If we are making the logo horizontal at medium screen sizes, this can be removed